packages/cost/usage/getUsageProcessor.ts

```diff
@@ -0,0 +1,6 @@
+import { OpenAIUsageProcessor } from "./openAIUsageProcessor";
+import { IUsageProcessor } from "./IUsageProcessor";
+
+export function getUsageProcessor(provider: string): IUsageProcessor {
```

Comment:
Can you type the provider here? We should have it somewhere.

Also for now let's throw an error if we call this and the provider is not anthropic.